packages/react-devtools-shell/src/e2e/app.js

```diff
@@ -2,19 +2,19 @@
 
 // This test harness mounts each test app as a separate root to test multi-root applications.
 
-import {createElement} from 'react';
-import {
-  // $FlowFixMe Flow does not yet know about createRoot()
-  createRoot,
-} from 'react-dom';
-import ToDoList from '../app/ToDoList';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 
 const container = document.createElement('div');
 
 ((document.body: any): HTMLBodyElement).appendChild(container);
 
 // TODO We may want to parameterize this app
 // so that it can load things other than just ToDoList.
+const App = require('./apps/ListApp').default;
```

Comment:
import App from './apps/ListApp' ?

Reply:
This is a require b'c I'm planning on adding URL param support in the future to this page to switch between different apps. (We might want test-specific apps.)

Reply:
that's futurism